### ISSUE TITLE
Rebrand DAO Dashboard - Layout

### ISSUE THREAD

**Issue by peterjurco** [api3dao/api3-dao-dashboard#455]
Update the main styles of the DAO Dashboard's layout according to [rebranded designs](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=8676-107801&node-type=section&m=dev). This includes
- navigation
- menu (desktop and mobile)
- logo
- Connect button
- Connected user dropdown
- Footer
- Error reporting bar

**Comment by PrenaamD** [comment]
@peterjurco 
1. 
The implemented Error reporting bar is different than the design's
Implementation
![Image](https://github.com/user-attachments/assets/b1c6bb50-38f2-4e4f-a91f-6fa8ef4b0f20)
Design
![Image](https://github.com/user-attachments/assets/b8c64c21-474c-4e5d-a44e-0d95f915f375)

2. 
In the account dropdown the Disconnect button color seems different from the design
Implementation
![Image](https://github.com/user-attachments/assets/63f58fb8-fac1-4e96-9e7a-f2b2785e4ece)
Design
![Image](https://github.com/user-attachments/assets/63c2a403-95b1-4614-ac73-308351eff672)

**Comment by peterjurco** [comment]
@PrenaamD Good observations. Both of these questions are being handled

1. I created a [new issue](https://github.com/api3dao/api3-dao-dashboard/issues/467#issue-2641311825) for the cookies consent because we need to add a whole new modal (which wasn't ready yet at the time I implemented it)

2. The issue is that in designs it is on the white background while we have a bit more gray background on the page so the dropdown is not visible. I asked about it Leandro and they are going to [address it](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen?node-id=8676-107934&m=dev#1024482533) (after that I will update it)
